shell/platform/windows/flutter_windows.cc

```diff
@@ -239,3 +240,33 @@ void FlutterDesktopMessengerSetCallback(FlutterDesktopMessengerRef messenger,
                                         void* user_data) {
   messenger->dispatcher->SetMessageCallback(channel, callback, user_data);
 }
+
+// Texture support isn't implemented yet for Windows,
+// Please refer to https://github.com/flutter/flutter/issues/38601.
+FlutterDesktopTextureRegistrarRef FlutterDesktopGetTextureRegistrar(
+    FlutterDesktopPluginRegistrarRef registrar) {
+  std::cout << "Not yet implemented.\n";
```

Comment:
These messages need to give context; there's not stack associated with a `std::cout`. It should say something like "Texture support is not yet implemented."

Also, prefer `std::endl` to a literal `\n`, so it flushes.